client/web/src/site-admin/SiteAdminConfigurationPage.tsx

```diff
@@ -526,3 +543,142 @@ class SiteAdminConfigurationContent extends React.Component<Props, State> {
         this.siteReloads.next()
     }
 }
+
+interface SiteConfigurationChangeProps {}
+
+const SiteConfigurationChangeListPage: React.FunctionComponent<SiteConfigurationChangeProps> = () => {
+    const { connection, loading, error, refetch, ...paginationProps } = usePageSwitcherPagination<
```

Comment:
`refetch` doesn't seem to be used
```suggestion
    const { connection, loading, error, ...paginationProps } = usePageSwitcherPagination<
```

Reply:
`loading` and `error` too, but I think they should be used to display `ConnectionLoading` and `ConnectionError` components respectively

Reply:
Nice catch. 👌